doc/sphinx/reference/kinetics/reaction-rates.md

```diff
@@ -106,6 +106,72 @@ reactions:
+
+Note that $n_{\rm elec}$ and $\Delta \phi$ both have a sign, so the terms "increase" and "decrease" are relative; the forward barrier height might increase by a negative amount (i.e. decrease), for instance.
+
+From transition state theory, the forward and reverse reaction rates are therefore calculated as:
+
+$$ R_f = k_f\exp\left(-\frac{\beta n_{\rm elec}F\Delta\phi}{RT} \right)\Pi_k C_{{\rm ac},\,k}^{\nu_k^\prime}\$$
+
+and
+
+$$ R_r = k_r\exp\left(\frac{\left(1-\beta\right)n_{\rm elec}F\Delta\phi}{RT} \right)\Pi_k C_{{\rm ac},\,k}^{\nu_k^{\prime\prime}},$$
+
+respectively, where $k_f$ and $k_r$ are the normal chemical rate coefficients in the absence of any electric potential difference (e.g., calculated using Arrhenius coefficients), $C_{{\rm ac},\,k}$ is the activity concentration of species $k$, $\nu_k^\prime$ and $\nu_k^{\prime\prime}$ are the forward and reverse stoichiometric coefficients, respectively, for species $k$ for this reaction, and $R$ and $T$ are the universal gas constant and temperature, respectively.
+
+Note that Cantera's actual software implementation looks quite different from the description above, which is meant solely to give a clearer understanding of the science behind Cantera's calculations.
+```{admonition} YAML Usage
+:class: tip
+- Electrochemical reactions only occur at phase boundaries and therefore use the standard [``interface``](sec-yaml-interface-Arrhenius) reaction rate implementation.
+- Charge transfer is automatically detected, and $n_{\rm elec}$ automatically calculated.  If no value for `beta` is provided, an [``electrochemical``](sec-yaml-electrochemical-reaction) reaction assumes a default of ``beta = 0.5``.
+```
+
+(sec-butler-volmer)=
+### The Butler-Volmer Form
+
+Cantera's electrochemical reaction rate calculation is equivalent to the commonly-used Butler-Volmer rate form. In Butler-Volmer, the net rate of progress, $R_{\rm net} = R_f - R_r$, can be written as:
+
+$$ R_{\rm net} = \frac{i_\circ}{n_{\rm elec}F}\left[\exp\left(-\frac{\beta n_{\rm elec}F\eta}{RT} \right) - \exp\left( \frac{\left(1-\beta\right)n_{\rm elec}F\eta}{RT}\right) \right]$$
+
+where the kinetic rate constant $i_\circ$ is known as the "exchange current density" and $\eta$ the "overpotential" -- the difference between the actual electric potential difference and that which would set the reaction to equilibrium:
+
+$$ \eta = \Delta\phi - \Delta\phi_{\rm equil}$$
+
+To convert between the two forms, the exchange current density varies with the chemical state and can be calculated as:
+
+$$i_\circ = n_{\rm elec}Fk_f^{\left(1-\beta\right)}k_r^\beta\Pi_k C_{{\rm ac},\,k}^{\left(1-\beta\right)\nu_k^\prime}\Pi_k C_{{\rm ac},\,k}^{\beta\nu_k^{\prime\prime}}.$$
+
+
+````{admonition} YAML Usage
+:class: tip
+- One can explicitly provide an exchange current density, rather than the $k_f$ value, by setting the optional ``exchange-current-density-formulation`` field to ``true``.
```

Comment:
```suggestion
One can explicitly provide an exchange current density, rather than the $k_f$ value, by setting the optional ``exchange-current-density-formulation`` field to ``true``.
```